src/analysis/retail/druid/restoration/modules/spells/Dragonflight2pc.tsx

```diff
@@ -0,0 +1,93 @@
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import { calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import SPELLS from 'common/SPELLS';
+import { TIERS } from 'game/TIERS';
+import Events, { HealEvent } from 'parser/core/Events';
+import BoringValueText from 'parser/ui/BoringValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import HIT_TYPES from 'game/HIT_TYPES';
+import { formatNumber } from 'common/format';
+import ItemPercentHealingDone from 'parser/ui/ItemPercentHealingDone';
+
+const TWO_PIECE_CRIT_BONUS = 0.08;
+
+const TWO_PIECE_SPELLS = [
+  SPELLS.REJUVENATION,
+  SPELLS.REJUVENATION_GERMINATION,
+  SPELLS.LIFEBLOOM_BLOOM_HEAL,
+  SPELLS.LIFEBLOOM_HOT_HEAL,
+  SPELLS.TRANQUILITY_HEAL,
+  SPELLS.EFFLORESCENCE_HEAL,
+  SPELLS.WILD_GROWTH,
+];
+
+/**
+ **Dragonflight 2PC**
+ *
+ * Rejuvenation, Lifebloom, Wild Growth, Efflorescence, and Tranquility chance to critically heal is increased by 8%.
+ */
+class Dragonflight2pc extends Analyzer {
```

Comment:
I would recommend something like `T292pc`, `2pcT29`, or `VotI2pc` instead, as `Dragonflight2pc` is a bit too vague.